app/src/scripts/convert.js

```diff
@@ -82,6 +82,19 @@ function convertToWebm(opts) {
   });
 }
 
+// Should be similiar to the Gif generation
+function convertToApng(opts) {
+  return Promise.resolve().then(() => {
+    return convert(opts.outputPath, opts, [
+      '-i', opts.filePath,
```

Comment:
This command (testing)
`ffmpeg -i test.mp4 -filter_complex fps=14,scale=1024:1024:flags=lanczos[x] tests.apng`
returns the following error
`Filter scale has an unconnected output`. According to ffmpeg Filters [docs](https://ffmpeg.org/ffmpeg-all.html#Filtering) filter_complex should be used for multiple outputs. Is it not enough to use `-vf` instead of -filter_complex ?
I've tried your settings just with `-vf`and it works in the command-line and in the electron app.
`ffmpeg -i test.mp4 -vf fps=14,scale=1024:1024:flags=lanczos[x] -plays 0 test_convert.apng`